### PR TITLE
ISSUE_HELP.md: improve terminology

### PR DESCRIPTION
**Pull request by Andersson007**
##### SUMMARY
ISSUE_HELP.md: improve terminology

##### ISSUE TYPE
- Docs Pull Request

##### COMPONENT NAME
ISSUE_HELP.md

### CODE REVIEW

File: ISSUE_HELP.md [hunk 1]
```diff
@@ -158,8 +158,8 @@ Command | Scope | Allowed | Description
...
 **<a name="cmd-shipit">shipit</a>** | pull requests | maintainers | If you approve the code in this pull request, use this command to have it merged. Note that Github `Approve` pull request status is ignored. Nonetheless `shipit` in review summary of commented or approved review is taken in account. In place of `shipit`, `+1` and `LGTM` can be used too. Note that these commands must not be surrounded by any character, spaces excepted.
-**<a name="cmd-add-label">+label</a>** | issues pull requests | staff maintainers | Add a whitelisted label. See [When to use label commands](#when-to-use-label-commands).
-**<a name="cmd-remove-label">-label</a>** | issues pull requests | staff maintainers | Remove a whitelisted label. See [When to use label commands](#when-to-use-label-commands).
+**<a name="cmd-add-label">+label</a>** | issues pull requests | staff maintainers | Add a [supported label](#labels). See [When to use label commands](#when-to-use-label-commands).
+**<a name="cmd-remove-label">-label</a>** | issues pull requests | staff maintainers | Remove a label. See [When to use label commands](#when-to-use-label-commands).
```
Comment (felixfontein) [inline ISSUE_HELP.md:162]: Maybe also here:
```suggestion
**<a name="cmd-remove-label">-label</a>** | issues pull requests | staff maintainers | Remove a supported label. See [When to use label commands](#when-to-use-label-commands).
```

Comment (Andersson007) [reply ISSUE_HELP.md]: @felixfontein , sure (yes, i think the link is a bit extra but the adjective is good), thanks, committed:)

**Review by felixfontein — APPROVED** [review]

**Review by gundalow — APPROVED** [review]
Good spot, thank you. 
Could you possibly make the same change in https://github.com/ansible/ansibullbot/

**Comment by Andersson007** [comment]
@gundalow sure:)

**Comment by Andersson007** [comment]
done https://github.com/ansible/ansibullbot/pull/1496

**Pull request merged**